### ISSUE TITLE
Spinning cube example in README crashes latest Android Chrome

### ISSUE THREAD

**Issue by kungfooman** [playcanvas/engine#4821]
I just updated Chrome on Android to the latest version (Chrome on Android | 107.0.5304.91 | 2022-10-28) and now it doesn't even want to render a cube anymore, because this vertex shader contains some issue:

```js
#extension GL_OES_standard_derivatives : enable
#define SHADER_NAME LitShader
varying vec3 vPositionW;
varying vec3 vNormalW;

attribute vec3 vertex_position;
attribute vec3 vertex_normal;
attribute vec4 vertex_tangent;
attribute vec2 vertex_texCoord0;
attribute vec2 vertex_texCoord1;
attribute vec4 vertex_color;

uniform mat4 matrix_viewProjection;
uniform mat4 matrix_model;
uniform mat3 matrix_normal;

vec3 dPositionW;
mat4 dModelMatrix;
mat3 dNormalMatrix;


mat4 getModelMatrix() {
    return matrix_model;
}

vec4 getPosition() {
    dModelMatrix = getModelMatrix();
    vec3 localPos = vertex_position;




    vec4 posW = dModelMatrix * vec4(localPos, 1.0);
    dPositionW = posW.xyz;

    vec4 screenPos;
    screenPos = matrix_viewProjection * posW;


    return screenPos;
}

vec3 getWorldPosition() {
    return dPositionW;
}


vec3 getNormal() {
    dNormalMatrix = matrix_normal;

    vec3 tempNormal = vertex_normal;



    return normalize(dNormalMatrix * tempNormal);
}


void main(void) {
    gl_Position = getPosition();
   vPositionW    = getWorldPosition();
   vNormalW = getNormal();

}

```

![image](https://user-images.githubusercontent.com/5236548/200184066-b56af1fe-2749-4618-8d17-ceea3b259996.png)

Anyone else run into this issue?

**Comment by mvaligursky** [comment]
We've seen shader compilation errors with no details when the WebGl context cannot be created / is lost. On Mac that usually means the computer needs to be rebooted, or at least the browser restarted. I have not seen that on Android yet, but try the restart? Also try some other webgl applications to see if those are affected as well.

**Comment by mvaligursky** [comment]
By the way, I have exactly the same Chrome version on Android 13, and examples at https://playcanvas.github.io/ run without a problem for me.

**Comment by kungfooman** [comment]
> Also try some other webgl applications to see if those are affected as well.

Thank you, I tried an ElementComponent example and the UI rendered fine, but the 3D cube crashed it immediately (e.g. after adding the cube over Android/Chrome-devtools).

My Android OS was rather old, so now I updated to lineage-18.1-20221101-nightly-flox-signed and... no problem so far with the shipped browser (looks like Chromium, but can't find a way to print the version so far...). Running out of time to test latest Chrome again today.

So I guess it's fair to say the OS was just too old for latest Chrome and for some reason the vertex shader failed to compile while crashing the context.
